src/pages/_app.page.tsx

```diff
@@ -1,7 +1,10 @@
+import { ContentfulLivePreviewProvider } from '@contentful/live-preview/react';
 import { appWithTranslation } from 'next-i18next';
 import './utils/globals.css';
 import type { AppProps } from 'next/app';
 import { Urbanist } from 'next/font/google';
+import '@contentful/live-preview/style.css';
```

Comment:
Being super picky here but maybe it's nice to have the styles at the bottom of the imports?